tests/cbf_config_input/sonic_version.yml

```diff
@@ -0,0 +1,8 @@
+build_version: 'master.487-a98cf221'
```

Comment:
may be we can reuse the exsiting one.. CBF, QoS are close related anyway.

Reply:
reused the qos one